src/css/profile/mobile/common/radio.less

```diff
@@ -25,14 +25,15 @@ input[type="radio"].ui-radio {
 		height: 100%;
 		width: 100%;
 		bottom: 0;
-		background-color: var(--control-active-color);
+		background-color: var(--control-inactive-color);
```

Comment:
How about the opacity?

Reply:
Right, Need to apply the opacity when inactive state.

Reply:
done.

Reply:
It would be good to use the @fade, but I need a check whether fade is working correctly on emulator.